test/jest/sideloadedKit.spec.ts

```diff
@@ -86,4 +85,25 @@ describe('MPSideloadedKit', () => {
             expect(filterDictionary.userAttributeFilters).toEqual(expectedResult);
         });
     });
+
+    describe('#MPSideloadedKit Integration Tests', () => {
```

Comment:
Move this into a separate integration test file so we can run independently.

Reply:
Added sus eng - https://mparticle-eng.atlassian.net/browse/SQDSDKS-5424